packages/framework/src/shared/dto.ts

```diff
@@ -55,3 +56,19 @@ export interface EnsembleConfigYAML {
   environmentVariables?: { [key: string]: unknown };
   secretVariables?: { [key: string]: unknown };
 }
+
+export interface FirebaseFontDTO {
+  readonly fontFamily: string;
+  readonly publicUrl: string;
+  readonly fontWeight: string;
+  readonly fontStyle: string;
+}
+
+export interface FontDTO {
```

Comment:
Just keep `EnsembleFontModel` and remove this, rename the FirebaseFontDTO to FontDTO. Convert things later so models reference models and dtos reference dtos, etc.

DTO = Data Transfer Object, it represents the data over the wire.